src/sysext/sysext.c

```diff
@@ -1839,8 +1839,10 @@ static int image_discover_and_read_metadata(
 
         HASHMAP_FOREACH(img, images) {
                 r = image_read_metadata(img, image_class_info[image_class].default_image_policy);
-                if (r < 0)
-                        return log_error_errno(r, "Failed to read metadata for image %s: %m", img->name);
+                if (r < 0) {
+                        log_debug_errno(r, "Failed to read metadata for image %s, ignoring: %m", img->name);
```

Comment:
I think we should raise this to LOG_NOTICE level. i.e. it's definitely interesting to report this even if it doesn't quite deserve a warning.

Reply:
can we provide more helpful log messages here, btw? i.e. might make sense to say what specifically failed if we know it, i.e. bad signature or bad image format or so.

(but it's fine to merge a simple version for now, I totally see that generated and picking up proper error messages is a bit more work)

Reply:
I'd make it a warning at least, if not an error